project/src/main/scala/licenses/DependencySummary.scala

```diff
@@ -163,10 +174,12 @@ object ReviewedSummary {
               case Some(includedLicense) =>
                 val licenseContent = IO.read(defaultPath.toFile)
                 if (licenseContent.strip != includedLicense.content) {
-                  warnings.append(
-                    s"A license file was discovered in $name that is different " +
-                    s"from the default license file that is associated with its " +
-                    s"license ${dep.information.license.name}."
+                  diagnostics.append(
+                    Diagnostic.Problem(
+                      s"A license file was discovered in $name that is different " +
+                      s"from the default license file that is associated with its " +
+                      s"license ${dep.information.license.name}, but a custom license was not expected. If this custom license should override the default one, create a `custom-license` config file. If both files are expected to be included, create an empty `default-and-custom-license` file."
```

Comment:
```suggestion
                      s"""|A license file was discovered in $name that is different 
                            |from the default license file that is associated with its 
                            |license ${dep.information.license.name}, but a custom license was not expected. If this custom license should override the default one, create a `custom-license` config file. If both files are expected to be included, create an empty `default-and-custom-license` file.""".stripMargin
```

Reply:
This will add newlines into the string, whereas before the string was a single line.